spec/namespaces/search_pipeline.yaml

```diff
@@ -23,12 +24,26 @@ paths:
       externalDocs:
```

Comment:
@nhtruong @Xtansia `PUT /_search/pipeline/my_pipeline` endpoint [creates](https://opensearch.org/docs/latest/search-plugins/search-pipelines/creating-search-pipeline/) and [updates](https://opensearch.org/docs/latest/search-plugins/search-pipelines/creating-search-pipeline/#updating-a-search-pipeline) search_pipeline. 
So, should we name it `search_pipeline.put` instead of `search_pipeline.create`?

Reply:
Yes I think so, it makes more sense semantically. Especially when things such as the _doc APIs set the precedent that "create" means fail if the item already exists.

Reply:
agree with @Xtansia